lib/chewy/rake_helper.rb

```diff
@@ -203,6 +203,22 @@ def all_indexes
         Chewy::Index.descendants - [Chewy::Stash::Journal, Chewy::Stash::Specification]
       end
 
+      # Reindex data from source index to destination index
+      #
+      # @example
+      #   Chewy::RakeHelper.reindex(only: ['users_index', 'cities_index']) reindex data from 'users_index' index to 'cities_index'
+      #
+      # @param only [Array<Chewy::Index, String>, Chewy::Index, String] indexes to reindex
+      def reindex(source:, dest:, output: $stdout)
+        raise ArgumentError, 'Please specify the source index and the destination index' unless source || dest
```

Comment:
```suggestion
        raise ArgumentError, 'Please specify the source index and the destination index' unless source && dest
```

Because of this, let's add tests covering both cases. :) Right now we have a test for skipped destination, let's have a similar test for skipped source.

Reply:
Hold on, hold on :D Actually the whole thing is redundant, you'll automatically get ArgumentError, as these are required keyword arguments. :)